pickATrip_django_apps/admin.py

```diff
@@ -23,11 +24,20 @@ def __init__(self, general_model, admin_site):
         super(ListAdminMixin, self).__init__(general_model, admin_site)
 
 
+def is_related_to_social_signing(suspected_model):
```

Comment:
can you please elaborate more what the problem is and how does this function solve it?

Reply:
The automation of the model registration suppose to register to admin only models that haven't been registered already, regarding the 4, 5 models provided by Django that are allowing social account logins (like google sign in) are already registered by Django and then the code tries to register it although it already has been registered, and this causes an exception. I couldn't find a solution so I enforced the automation part to avoid registering the 4, 5 problematic models that have been registered by Django to avoid the problem using an ugly if statement that assuring the model that is registered is not one of these problematic ones (might have something to do with the fact the social account models are registered differently because it happened automatically after adding the new settings to support social login)